src/platform_impl/linux/x11/window.rs

```diff
@@ -1699,6 +1707,21 @@ impl UnownedWindow {
             .expect_then_ignore_error("Failed to set WM hints");
     }
 
+    #[inline]
+    pub fn request_activation_token(&self) -> Result<AsyncRequestSerial, NotSupportedError> {
+        // Get the activation token and then put it in the event queue.
+        let token = self
```

Comment:
But shouldn't we call `new` with our startup sequence? And then I'd assume other client we're launching must do the `remove`?

From the spec

```

Some details of the startup sequence:

 - "new" and "change" messages are sent by the launcher code

 - the launchee code is responsible for sending a "remove" 
   message to end the launch sequence, unless the WMCLASS 
   key was set.

 - the "new" message must be the first message. Other message 
   types should be ignored by all clients unless those clients
   have seen a "new" message with the same ID.

 - "change" messages can be sent at any time between "new" and
   "remove"

```

I'm not entirely sure how reliable that should be though, and what will happen if client will fail to remove, but I guess nothing that bad...

Reply:
Changed to do this.